packages/conferer/src/Conferer/Source/InMemory.hs

```diff
@@ -15,30 +15,64 @@ import Data.Text (Text)
 
 import Conferer.Source
 
+newtype ExplainNotFound = ExplainNotFound (Key -> String)
+newtype ExplainSettedKey = ExplainSettedKey (Key -> String)
```

Comment:
💭 `ExplainSettedKey` is for when a key was set with an invalid value? I think the name could be better. What about, `ExplainInvalidValue` ? 🤔

Reply:
the thing is that it really doesn't do that, the point of explainSettedKey is being able to refer to a setted key, for example instead of saying `the key "some.key"`, in the cli source it would get translated to `cli arg "--some.key"`, I'll add some docs to the Source typeclass